libkbfs/folder_branch_ops.go

```diff
@@ -6193,6 +6466,10 @@ func (fbo *folderBranchOps) SyncFromServer(ctx context.Context,
 	if err := fbo.fbm.waitForSyncCacheCleans(ctx); err != nil {
 		return err
 	}
+	if err := fbo.partialSyncs.Wait(ctx); err != nil {
```

Comment:
Same question here--we have to wait for the whole partial sync?

Reply:
Yes, but this method is only used by tests and git operation (which wouldn't have partial syncing).